src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+
+public class ColorSupplier {
+    private final List<Color> availableColors = Arrays.asList(Color.values());
+    private final Random random = new Random();
+
+    public Color getRandomColor() {
+        int randomIndex = random.nextInt(availableColors.size() - 1);
```

Comment:
The `random.nextInt(availableColors.size() - 1)` call should be `random.nextInt(availableColors.size())` to ensure that the last color in the list can also be selected. The current implementation excludes the last color.